services/admin/src/server.ts

```diff
@@ -206,9 +206,9 @@ export function buildApp({ workspace }: { workspace: Workspace }): Application {
     { electionId: Id },
     Admin.DeleteCvrFileResponse,
     Admin.DeleteCvrFileRequest
-  >('/admin/elections/:electionId/cvr-files', (request, response) => {
+  >('/admin/elections/:electionId/cvr-files', async (request, response) => {
     const { electionId } = request.params;
-    store.deleteCastVoteRecordFiles(electionId);
+    await store.deleteCastVoteRecordFiles(electionId);
```

Comment:
If `transaction` is made sync or async depending on the callback then this would no longer need to be async, right?